wgpu/src/backend/wgpu_core.rs

```diff
@@ -862,6 +862,15 @@ impl dispatch::InstanceInterface for ContextWgpuCore {
             Err(err) => self.handle_error_fatal(err, "Instance::poll_all_devices"),
         }
     }
+
+    #[cfg(feature = "wgsl")]
+    fn wgsl_language_features(&self) -> std::collections::HashSet<crate::WGSLLanguageFeatures> {
+        use wgc::naga::front::wgsl::WGSLLanguageExtension;
+        WGSLLanguageExtension::all()
+            .iter()
+            .map(|_| unimplemented!())
```

Comment:
I feel like this should be implemented, but because the enum set is empty, you can leave it as an empty match.

This way when naga grows more, it will throw an error and force people to fix it.

Reply:
You can't have empty match unfortunately, so you always get
```rust
match wle {
                _ => todo!(),
            }
```
which clippy wants to transform into `todo!()`. So I think this is the best way for now, until enum will not be empty (I work https://github.com/gfx-rs/wgpu/issues/6192 afterwards and fix that).

Reply:
Huh? https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=91b5b0acd933e1b90bb7890bc0b6cdf2 seems to compile

Oh because it's non-exhaustive? Probably not worth making it non-exhaustive then.

Reply:
I think we may have gotten race conditioned here 😅

Reply:
Interesting, `&WGSLLanguageExtension` is still considered non-exhaustive: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=3f224420a90f050c3888f704c84df9ad, but `WGSLLanguageExtension` is not anymore.

Reply:
Now that's a spicy meatball